vpt/processors/gaze_engagement_estimator/__init__.py

```diff
@@ -0,0 +1,15 @@
+from rx import Observable
+from rx.subject import Subject
+
+from nodes import ProcessorBase
+import numpy as np
+
+
+class GazeEngagementEstimator(ProcessorBase[np.ndarray]):
+    subj = Subject()
+
+    def __init__(self, rotation_vector_source: ProcessorBase[np.ndarray]):
```

Comment:
The type should be `SourceBase`. This class shouldn't worry if the source is a processor or an actual source as long as it can take data from it